src/globals.js

```diff
@@ -44,6 +44,7 @@ const DefaultPrefs = {
 	HighlightStyles_ObscuredRect:  'fill: rgba(127,127,127,.10); stroke: rgba(127,127,127,.60); stroke-width: 1px;',
 
 	OpenTabsAtEndOfTabBar: false,
+	CopyLinksToClipboard: false,
```

Comment:
To address @cpriest comments you could create 2 integers, `DefaultAction`, `ModifierAction` and add something like the following comment
```Javascript
/* 
0: Open links
1: Copy links
2: To be continued
*/